test/lib/y2firewall/clients/installation_finish_test.rb

```diff
@@ -69,26 +69,30 @@
     let(:api) do
       instance_double(Y2Firewall::Firewalld::Api, remove_service: true, add_service: true)
     end
+    let(:installation) { true }
 
     before do
       allow(proposal_settings).to receive("enable_firewall").and_return(enable_firewall)
       allow(firewalld).to receive("api").and_return(api)
       allow(firewalld).to receive("enable!")
       allow(firewalld).to receive("disable!")
+      allow(Yast::Mode).to receive(:installation).and_return(installation)
       allow(proposal_settings).to receive("open_ssh").and_return(false)
     end
 
-    it "enables the firewalld service if enabled in the proposal" do
-      allow(proposal_settings).to receive("enable_firewall").and_return(true)
-      expect(firewalld).to receive("enable!")
+    context "during an installation" do
+      it "enables the firewalld service if enabled in the proposal" do
+        allow(proposal_settings).to receive("enable_firewall").and_return(true)
+        expect(firewalld).to receive("enable!")
```

Comment:
np: Using a string looks strange. What about simply using a symbol (`:enable!`)? It is what we usually do.

Reply:
Yep, that is true. Will modify it.

Reply:
You have not modified it, right? :)

Reply:
Now, it is definitely modified :stuck_out_tongue_closed_eyes: